### PR TITLE
BS-13283, move content in an other object because DISCTINCT on table …

### PR DESCRIPTION
**Pull request by baptistemesta**
…with CLOB is not supported in oracle

### CODE REVIEW

File: bonita-integration-tests/bonita-query-tests/src/test/resources/testContext.xml [hunk 1]
```diff
@@ -7,7 +7,8 @@
             http://www.springframework.org/schema/beans 
             http://www.springframework.org/schema/beans/spring-beans-3.1.xsd">
 
-    <jdbc:embedded-database id="dataSource" type="H2" />
+    <jdbc:embedded-database id="dataSource" type="H2" >
+    </jdbc:embedded-database>
```
Comment (educhastenier) [inline bonita-integration-tests/bonita-query-tests/src/test/resources/testContext.xml]: why did you change this bean like this?

Comment (baptistemesta) [inline bonita-integration-tests/bonita-query-tests/src/test/resources/testContext.xml]: it's a mistake, i'll revert it

File: bpm/bonita-core/bonita-process-definition/bonita-process-definition-impl/src/main/resources/org/bonitasoft/engine/core/process/definition/model/impl/hibernate/process.definition.hbm.xml [hunk 2]
```diff
@@ -6,7 +6,27 @@
 		<filter-param name="tenantId" type="long" />
 	</filter-def>
 
-	<class name="SProcessDefinitionDeployInfoImpl" table="process_definition">
+	<class name="SProcessDefinitionDeployInfoImpl" table="process_definition" polymorphism="explicit">
```
Comment (educhastenier) [inline bpm/bonita-core/bonita-process-definition/bonita-process-definition-impl/src/main/resources/org/bonitasoft/engine/core/process/definition/model/impl/hibernate/process.definition.hbm.xml]: Cannot you have inheritance between the 2 Hibernate beans?
